datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/io/Encoding.kt

```diff
@@ -152,14 +181,13 @@ object MultibaseHelper {
 
     @Deprecated("Dependency of calcEncPublicKeyCoords - Use [multiKeyGetKty] instead ")
```

Comment:
if this method is private, why not migrate it to the mentioned other method?

Reply:
Fixed